ruma-common/src/push.rs

```diff
@@ -80,6 +85,97 @@ impl Ruleset {
             AnyPushRule::Sender(r) => self.sender.insert(r),
         }
     }
+
+    /// Get the push actions that apply to this event.
+    ///
+    /// Returns `None` if no push action applies.
```

Comment:
```suggestion
    /// Returns `None` if no push condition applies.
```

Reply:
I would go for "rule" instead of condition. Because a condition can apply but not the whole rule

Reply:
Sure.